.vscode/launch.json

```diff
@@ -31,6 +31,21 @@
       "sourceMaps": true,
       "outFiles": []
     },
+    {
+      "name": "Debug current folder test",
```

Comment:
This adds a way to attach a debugger to the jest instance that will run the tests in your current folder.